inception-html-editor/src/main/java/de/tudarmstadt/ukp/inception/htmleditor/annotatorjs/resources/annotator-full.js

```diff
@@ -13,13 +12,14 @@
 
 //
 
+
 // Generated by CoffeeScript 1.6.3
```

Comment:
This is an automatically generated file and should therefore not be changed manually.